server/Controllers/LoginController.cs

```diff
@@ -44,9 +46,14 @@ public LoginController(IUserRepository userRepository, IConfiguration config)
         /// <returns>The result of the login.</returns>
         [AllowAnonymous]
         [HttpPost]
-        public IActionResult Login([FromBody] LoginBody body)
+        public ActionResult Login([FromBody] LoginBody body)
         {
-            IActionResult response = Unauthorized();
+            const string errormsg = "The email address and/or the password did not match.";
```

Comment:
Wellicht vertalen naar NL 😁

Reply:
Dit wordt een string die i8n zal kunnen vertalen, dit zal in een latere stage van development gedaan worden.

Reply:
Joe